spec/requests/graphql/mutations/create_dynamic_field_category_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::CreateDynamicFieldCategory, type: :request do
+  describe '.resolve' do
```

Comment:
When doing a request test, we want to think of this as a test of the external API rather than a test of a particular method (like `resolve`), so you should be able to remove the `describe '.resolve' do` block and just run the stuff inside.